README.md

```diff
@@ -1 +1,92 @@
-# blindnet-sdk-java-server
+<h1 align="center">
+  blindnet devkit<br />
+  Java Server SDK
+</h1>
+
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163277439-edd00509-1d1b-4565-a0d3-49057ebeb92a.png#gh-light-mode-only" height="80" /></p>
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163549893-117bbd70-b81a-47fd-8e1f-844911e48d68.png#gh-dark-mode-only" height="80" /></p>
+
+<p align="center">
+  <strong>Java Server SDK for authentication tokens generation and user management</strong>
+</p>
+
+<p align="center">
+  <a href="https://blindnet.dev"><strong>blindnet.dev</strong></a>
+</p>
+
+<p align="center">
+  <a href="https://blindnet.dev/docs">Documentation</a>
+  &nbsp;•&nbsp;
+  <a href="https://github.com/blindnet-io/sdk-php-server/issues">Submit an Issue</a>
+  &nbsp;•&nbsp;
+  <a href="https://join.slack.com/t/blindnet/shared_invite/zt-1arqlhqt3-A8dPYXLbrnqz1ZKsz6ItOg">Online Chat</a>
+  <br>
+  <br>
+</p>
+
+## About
+
+This is the Java [server-side SDK](https://blindnet.dev/docs/glossary#server-side-sdk) component of [blindnet devkit][devkit] allowing you to:
+
+- Generate [authentication tokens](https://blindnet.dev/docs/glossary#authentication-token) to allow users to access blindnet.
+- Manage application users and data on blindnet.
+
+## Get Started
+
+:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.
+
+## Installation
+
+<!-- FIXME: add install steps -->
+Use [{package manager or plateform}][install-tool] to install {project's name}:
+
+```bash
+{install-command}
+```
+
+<!-- FIXME: add API Reference
+## Usage
+📑 The API reference of blindnet devkit Node.js Server SDK is available on [blindnet.dev](https://docs.blindnet.io/docs/api_reference/server/node.js/latest).
+-->
```

Comment:
TODO @blindnet-io/engineering

Reply:
There is no API reference for Java SDK.

Reply:
1. You still need to complete the "install steps" section
2. There should be an API reference. Why didn't you add one? (we'll address that in another issue and PR later)
3. Only the person who made a review comment should resolve it, to make sure it has really been addressed.